packages/web-react/src/components/Button/README.md

```diff
@@ -35,19 +35,20 @@ import { RouterLink } from 'react-router-dom';
-| `name`        | `string`                                                                                  | -          | no       | For use a button as a form data reference            |
-| `onClick`     | `string`                                                                                  | `null`     | no       | JS function to call on click                         |
-| `ref`         | `ForwardedRef<HTMLButtonElement>`                                                         | -          | no       | Button element reference                             |
-| `size`        | [Size dictionary][dictionary-size]                                                        | `medium`   | no       | Size variant                                         |
-| `type`        | `string`                                                                                  | `button`   | no       | Type of the Button                                   |
+| Prop name     | Type                                                                                      | Default    | Required | Description                                                                |
+| ------------- | ----------------------------------------------------------------------------------------- | ---------- | -------- | -------------------------------------------------------------------------- |
+| `children`    | `ReactNode`                                                                               | `null`     | no       | Content of the Button                                                      |
+| `color`       | [Action Color dictionary][dictionary-color], [Emotion Color dictionary][dictionary-color] | `primary`  | no       | Color variant                                                              |
+| `elementType` | `ElementType`                                                                             | `'button'` | no       | Type of element                                                            |
+| `isBlock`     | `bool`                                                                                    | `false`    | no       | Span the element to the full width of its parent                           |
+| `isDisabled`  | `bool`                                                                                    | `false`    | no       | If true, Button is disabled                                                |
+| `isLoading`   | `bool`                                                                                    | `false`    | no       | If true, Button is in a loading state, disabled and the Spinner is visible |
+| `isSquare`    | `bool`                                                                                    | `false`    | no       | If true, Button is square, usually only with an icon                       |
```

Comment:
Unify naming across packages and Button components: https://github.com/lmc-eu/spirit-design-system/pull/762#discussion_r1157195662.
```suggestion
| `isSquare`    | `bool`                                                                                    | `false`    | no       | If true, Button is square, usually only with an Icon                       |
```